_includes/manuals/3.1/1.2_release_notes.md

```diff
@@ -24,7 +24,291 @@ Existing installations should start thinking about a migration plan.
+* Fix test failures in plugins \- [#33560](https://projects.theforeman.org/issues/33560)
+* Introduce skip pattern for ACL tests \- [#33122](https://projects.theforeman.org/issues/33122)
+* capybara 3.32.1 breaks ReportTemplateJSIntegrationTest \- [#29481](https://projects.theforeman.org/issues/29481)
+
+#### Foreman - Trends
+* Remove statistics & trends API endpoints \- [#30504](https://projects.theforeman.org/issues/30504)
+
+#### Foreman - Users, Roles and Permissions
+* Unable to assign ansible roles to a host group via hammer/api with non-admin user \- [#33727](https://projects.theforeman.org/issues/33727)
+* Drop view_tasks permission in core \- [#33387](https://projects.theforeman.org/issues/33387)
+* Non-admin users can not list their Personal Access Tokens \- [#33371](https://projects.theforeman.org/issues/33371)
+
+#### Foreman - Web Interface
+* Host details tabs aren't clickable \- [#33805](https://projects.theforeman.org/issues/33805)
+* Add report and facts to host details kebab dropdown \- [#33714](https://projects.theforeman.org/issues/33714)
+* Add build review modal in the new host details page \- [#33696](https://projects.theforeman.org/issues/33696)
+* After creating new host group from help page, page doesn't redirect to main landing page. \- [#33637](https://projects.theforeman.org/issues/33637)
+* Remove experimental host wizard page \- [#33621](https://projects.theforeman.org/issues/33621)
+* "All audits" link in host details page doesn't work \- [#33617](https://projects.theforeman.org/issues/33617)
+* Adding a banner for the experimental host details page \- [#33597](https://projects.theforeman.org/issues/33597)
+* There should be buttons to link in between the old host page and the new host detail page \- [#33507](https://projects.theforeman.org/issues/33507)
+* The new host detail page audits card gets misaligned \- [#33505](https://projects.theforeman.org/issues/33505)
+* The new host detail page hostname should never be split to two lines \- [#33504](https://projects.theforeman.org/issues/33504)
+* Mac Address label should have capital letters in the word MAC on the new host detail page \- [#33503](https://projects.theforeman.org/issues/33503)
+* The host owner name should be expanded to the full width as it typically takes more than a one line \- [#33502](https://projects.theforeman.org/issues/33502)
+* New host detail page MAC should remain on one line if there's a long host group next to it \- [#33501](https://projects.theforeman.org/issues/33501)
+* Experimental host page kebab menu buttons do nothing \- [#33491](https://projects.theforeman.org/issues/33491)
+* Edit button in experimental host page doesn't work \- [#33490](https://projects.theforeman.org/issues/33490)
+* Add setting for redirecting to the new host details page  \- [#33484](https://projects.theforeman.org/issues/33484)
+* "Login Page Footer Text" Blocking Login Button on Satellite UI \- [#33481](https://projects.theforeman.org/issues/33481)
```

Comment:
This description should be updated - probably an issue that was cloned from BZ, but we need to modify it to match foreman